x-pack/solutions/security/plugins/security_solution/common/api/entity_analytics/entity_store/entities/delete_entity.schema.yaml

```
@@ -0,0 +1,42 @@
+openapi: 3.0.0
+info:
+  version: '2023-10-31'
+  title: Entity Store - Delete Single Entity
+paths:
+  /api/entity_store/entities/{entityType}/{entityId}:
+    delete:
+      x-labels: [ess, serverless]
+      x-codegen-enabled: true
+      operationId: DeleteSingleEntity
+      summary: Delete an entity in Entity Store
+      description: >
+        Delete a single entity in Entity Store.
+
+        The entity will be immediately gone from the latest index. 
```

This becomes official docs, maybe deleted is clearer than gone

```suggestion
        The entity will be immediately deleted from the latest index. 
```